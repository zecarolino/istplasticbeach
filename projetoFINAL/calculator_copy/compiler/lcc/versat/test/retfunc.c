int ret() { return 9; }
int main() { return ret(); }
