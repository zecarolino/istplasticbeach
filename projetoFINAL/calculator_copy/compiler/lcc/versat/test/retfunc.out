VCD info: dumpfile xtop.vcd opened for output.
9
Catched Trap at time  410 clock cycles:
